<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewOrdersDialog</class>
 <widget class="QDialog" name="ViewOrdersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(102, 16, 53);
}</string>
  </property>
  <widget class="QPushButton" name="infoBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: '#75103c';
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetBtn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: '#75103c';
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="timeButton">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>15</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: '#75103c';
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
border-radius: 10px;
}

QPushButton::hover {
background: '#75103c';
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="srchOrderBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simpified Jpan</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 19pt &quot;HP Simpified Jpan&quot;;
	border: 4px solid rgb(26, 24, 24);
	padding-left: 14px;
	background: rgb(117, 16, 60);
	color: rgb(209, 209, 209);
}</string>
   </property>
   <property name="text">
    <string>Search Order</string>
   </property>
   <property name="placeholderText">
    <string>Search Employee</string>
   </property>
  </widget>
  <widget class="QPushButton" name="srchButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: '#75103c';
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>971</width>
     <height>511</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
background-color: rgb(102, 16, 53);
border: 6px solid  '#1a1818';
gridline-color:  '#1a1818';
selection-background-color: rgb(84, 13, 43);
}
QHeaderView::section {
border-color: rgb(26, 24, 24);
color: rgb(209, 209, 209);
font: 20pt &quot;HP Simplified Jpan&quot;;
background-color: rgb(102, 16, 53);
height: 65px;
}

QHeaderView::section::hover {
background-color: rgb(102, 16, 53);
}

QTableView::item{
background-color: rgb(102, 16, 53);
color: rgb(209, 209, 209);
}

QTableView::item:hover {
background-color: rgb(84, 13, 43);
}
QTableView::item:selected {
background-color: rgb(102, 16, 53);
}</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>159</number>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="updtOrderBtn">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>230</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: '#75103c';
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
letter-spacing: 2px;
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rmvOrderBtn">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>420</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HP Simplified Jpan</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: '#75103c';
color: rgb(209, 209, 209);
border: 4px solid '#1a1818';
letter-spacing: 2px;
}

QPushButton::hover {
background-color: rgb(84, 13, 43);
}</string>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
